ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/Storage/LedgerDB/BackingStore/API.hs

```diff
@@ -0,0 +1,258 @@
+{-# LANGUAGE DataKinds            #-}
+{-# LANGUAGE DerivingVia          #-}
+{-# LANGUAGE FlexibleContexts     #-}
+{-# LANGUAGE NamedFieldPuns       #-}
+{-# LANGUAGE RankNTypes           #-}
+{-# LANGUAGE StandaloneDeriving   #-}
+{-# LANGUAGE UndecidableInstances #-}
+
+-- | The 'BackingStore' is the component of the
+-- 'Ouroboros.Consensus.Storage.LedgerDB.LedgerDB' implementation that stores a
+-- key-value map with the 'LedgerTable's at a specific point on the chain.
+--
```

Comment:
```suggestion
-- | The 'BackingStore' is the component of the
-- 'Ouroboros.Consensus.Storage.LedgerDB.LedgerDB' that stores a
-- key-value map with the 'LedgerTable's at a specific point on the chain.
--
```

In this paragraph, is "point" the same as our `Point` data type? If so we might want to replace the word point by a link to `Point`.

Reply:
It should say 'Slot'